p2p/net/conn.go

```diff
@@ -33,6 +33,8 @@ const (
 	Remote
 )
 
+const MessageQueueSize = 50
```

Comment:
Discuss with yosher what value it should be. Do we want to export it to config?

Reply:
@y0sher

Reply:
I can't calculate an accurate number, let's start with this and test. there are several queues in the way so everything should be buffered.